### PR TITLE
Package oraft.0.2.0

### PR DESCRIPTION
**Pull request by komamitsu**
### `oraft.0.2.0`
Raft consensus algorithm implemented in OCaml
Raft consensus algorithm implemented in OCaml

---
* Homepage: https://github.com/komamitsu/oraft
* Source repo: git+https://github.com/komamitsu/oraft.git
* Bug tracker: https://github.com/komamitsu/oraft/issues

---
:camel: Pull-request generated by opam-publish v2.1.0

### CODE REVIEW

File: packages/oraft/oraft.0.2.0/opam [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+]
+
+depends: [
+    "ocaml" {>= "4.05.0"}
+    "dune" {>= "2.0"}
+    "core" {>= "v0.9.0"}
+    "cohttp-lwt-unix"  {< "3.0.0"}
+    "yojson"
+    "ppx_deriving"
+    "ppx_deriving_yojson"
```
Comment (mseri) [inline packages/oraft/oraft.0.2.0/opam:22]:
```suggestion
    "ppx_deriving_yojson" {>= "3.6.0"}
```
Earlier versions are using ppxfind and don't play nice with recent dune and being mixed with ppxlib-based libraries

Comment (mseri) [reply packages/oraft/oraft.0.2.0/opam:22]: Will fix
```

#=== ERROR while compiling oraft.0.2.0 ========================================#
# context              2.1.2 | linux/x86_64 | ocaml-base-compiler.4.07.1 | pinned(https://github.com/komamitsu/oraft/archive/0.2.0.tar.gz)
# path                 ~/.opam/4.07/.opam-switch/build/oraft.0.2.0
# command              ~/.opam/opam-init/hooks/sandbox.sh build dune build -p oraft -j 31
# exit-code            1
# env-file             ~/.opam/log/oraft-19218-52b8d9.env
# output-file          ~/.opam/log/oraft-19218-52b8d9.out
### output ###
# File "lib/dune", line 5, characters 13-55:
# 5 |  (preprocess (pps ppx_deriving.std ppx_deriving_yojson)))
#                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
# Error: No ppx driver were found. It seems that ppx_deriving.std and
# ppx_deriving_yojson are not compatible with Dune. Examples of ppx rewriters
# that are compatible with Dune are ones using ocaml-migrate-parsetree, ppxlib
# or ppx_driver.
```

**Comment by kit-ty-kate** [comment]
Thanks

**Comment by komamitsu** [comment]
@mseri Thanks! I'll backport the change to upstream.

**Pull request merged**